sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/BroadcastExchangeExec.scala

```diff
@@ -45,9 +45,16 @@ import org.apache.spark.util.{SparkFatalException, ThreadUtils}
 trait BroadcastExchangeLike extends Exchange {
 
   /**
-   * The broadcast job group ID
+   * The broadcast run ID in job tag
    */
-  def runId: UUID = UUID.randomUUID
+  @deprecated("Use jobTag", "3.5.0")
```

Comment:
Is this annotation necessary?

Reply:
The reason is that, the semantics of runId lightly changed. Before it represents the job id, and now it is the unique id in job tag. But I'm fine to remove it if you think it is unnecessary.